packages/x-adapter/src/http-clients/fetch.http-client.ts

```diff
@@ -19,7 +19,7 @@ import { buildUrl, toJson } from './utils'
  *
  * @public
  */
-export const fetchHttpClient: HttpClient = async (
+export const rawFetchHttpClient: HttpClient = async (
```

Comment:
As it is an executable function and not a constant itself, I would name it as `fetchRawHttpClient`
WDYT to explain our intention as we talked about how the HTTP client should be? The "future" refactor with breaking-change. You can use a block in the doc `@remarks` for this.